### DISCUSSION TITLE
There are many get fans in China. Please improve the Chinese document. thank you

### DISCUSSION THREAD

**Discussion by TakahashiAiko** [jonataslaw/getx#1272] [Ideas]
If possible, you can find a partner in China to help you complete the translation .

**Comment by jonataslaw** [comment]
Apparently the GetX community in China is really growing. The repository has received about 20,000 daily visits from China, which is a high number for visits to a dart repository on github. I hope to be able to provide all necessary support to improve the experience and documentation of GetX for Chinese developers. If you have any suggestions on how we can improve on this, feel free to say so.
We have a community on discord that has a tab for Chinese speakers, but maybe discord is not so popular in china. I can create an official channel on another platform too, I think that there is nothing better for user learning than a centralized community.
